### ISSUE TITLE
Create Mongo 4.0.18 Docker Image to close security issues

### ISSUE THREAD

**Issue by strouja** [bitnami/bitnami-docker-mongodb#215]
**Description**

HI, I could not find a bitnami mongo docker image for version `4.0.18`  I did a quick search through the issues and did not see one created for this.  If possible can your team create one?  
Here are the security issues fixed as well as other bugs fixed in mongodb 4.0.18`
https://docs.mongodb.com/manual/release-notes/4.0-changelog/#id1

Ideally for if you can create it not on Oracle Linux but if that is easier to keep your 4.0.x images on Oracle Linux that is fine.

If I should not have created an issue for this and should have done something else just let me know and I apologize in advance.

Thanks for everything

**Steps to reproduce the issue:**

1. [First Step]
2. [Second Step]
3. [and so on...]

**Describe the results you received:**

**Describe the results you expected:**

**Additional information you deem important (e.g. issue happens only occasionally):**

**Version**

- Output of `docker version`:

```
(paste your output here)
```

- Output of `docker info`:

```
(paste your output here)
```

- Output of `docker-compose version` (if applicable):

```
(paste your output here)
```

**Additional environment details (AWS, VirtualBox, Docker for MAC, physical, etc.):**

**Comment by andresbono** [comment]
Hi @strouja, thanks for opening this issue. We no longer maintain the branch `4.0` of MongoDB. If you check the latest releases, you can find that only `4.2` is maintained and updated. Is it possible for you to upgrade to that newer branch?

**Comment by strouja** [comment]
@andresbono thanks for replying.  That was the 1st thing I tried to get the team to do.  They mentioned they are not able to upgrade to 4.2.x at the moment as our release in just about locked down and they won't have time to deal with any things breaking.  

If I were in charge I would just move to 4.2.x and the just get the teams to fix whatever breaks.  They did at least upgrade from `4.0.11` to `4.0.16` and got things to work.  A few things broke anyway b/c there was underlying switch from Debian 9 to Oracle Linux (just like red hat) and even though they thought they had all things in the image working on Oracle Linux, some things broke anyway that required us to modify the mongodb 4.0.16 docker image.

Thanks again, James

**Comment by andresbono** [comment]
Got it, @strouja. We are sticking to `4.2` branch and `debian-10` support. If add support for the older branches, we will post a message here.

**Comment by miguelaeh** [comment]
Hi @strouja ,
We have just released the image version 3.6 and 4.0 for debian-9 again.
https://hub.docker.com/r/bitnami/mongodb/tags
